test/commands/profiles/port.test.js

```diff
@@ -0,0 +1,128 @@
+const sinon = require('sinon');
+const { expect, test, constants } = require('@twilio/cli-test');
+const { Config, ConfigData } = require('@twilio/cli-core').services.config;
+
+const ProfilesPort = require('../../../src/commands/profiles/port');
+
+describe('commands', () => {
+  describe('profiles', () => {
+    describe('port', () => {
+      const setup = (
+        commandArgs = [],
+        { addProjects = 4, addProfiles = 3, portProfiles = true, removeCred = true, getCreds = true } = {},
+      ) =>
+        test
+          .do((ctx) => {
+            ctx.userConfig = new ConfigData();
+            for (let i = 1; i <= addProjects; i++) {
+              ctx.userConfig.addProject(`profile${i}`, constants.FAKE_ACCOUNT_SID);
+            }
+            for (let i = 1; i <= addProfiles; i++) {
+              ctx.userConfig.addProfile(
+                `configProfile${i}`,
+                constants.FAKE_ACCOUNT_SID,
+                '',
+                `${constants.FAKE_API_KEY}configProfile${i}`,
+                constants.FAKE_API_SECRET,
+              );
+            }
+          })
+          .twilioCliEnv(Config)
+          .twilioCreateCommand(ProfilesPort, commandArgs)
+          .stdout()
+          .stderr()
+          .do((ctx) => {
+            const fakePrompt = sinon.stub();
+            fakePrompt.onFirstCall().resolves({
+              confirmation: portProfiles,
+            });
+
+            ctx.testCmd.inquirer.prompt = fakePrompt;
+          })
+          .do((ctx) => {
+            ctx.testCmd.secureStorage.removeCredentials = () => {
+              return removeCred;
+            };
+          })
+          .do((ctx) => {
+            ctx.testCmd.secureStorage.getCredentials = (projectId) => {
+              return {
+                apiKey: getCreds === true ? constants.FAKE_API_KEY : 'error',
+                apiSecret: constants.FAKE_API_SECRET,
```

Comment:
Minor: I think here also we should return error like [this](https://github.com/twilio/twilio-cli/pull/266/files#diff-9db2227b42dcab0d81d2c37023a53795f40044c089ec700a1fad72a5d1c00fe3R50).

Reply:
Shouldn't really matter with the tests here, but updated `apiSecret` to reflect on 'error_message' instead for future ease! :)